en/upgrades/decimal-upgrade-to-2-03.rst

```diff
@@ -1,4 +1,6 @@
 Decimal Upgrade to 2.03
 =======================
 
+The next decimal upgrade of the IATI Standard (to version 2.03) will take place from Friday 12 May through to early October 2017. The full timetable below outlines each phase of the upgrade process and  includes information on when input is invited from the IATI Community. We anticipate the end of this process and the release of version 2.03 on **Tuesday 3 October 2017**.
 
+Specific proposals and public consultations in the upgrade process will be available in the `2.03 section on IATI’s online forum <https://discuss.iatistandard.org/c/standard-management/2-03-decimal-upgrade-proposals>`__ from the afternoon of Friday 12 May – you can read, comment or add a new proposal there.
```

Comment:
Yep let's keep this one and clear it out tomorrow/populate with the finalised proposals